File: src/repl/repl.py
from typing import List


class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:
        for i in range(0, len(nums)):
            if (nums[i] >= target):
                return i

        return len(nums)


# init solution
sol = Solution()

nums = [-1, 3, 5, 6]
target = 0

result = sol.searchInsert(nums, target)
print(result)

File: src/repl/repl.js

/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target) {

    const result = []
    // console.log(target)


    for (let i = 0; i < nums.length; i++) {
        const elementInner = nums[i];

        for (let j = i + 1; j < nums.length; j++) {
            const elementOuter = nums[j];

            if ((elementInner + elementOuter) === target) {
                result.push(i)
                result.push(j)
                return result
            }

        }

    }


    return result
};



const result = twoSum([2, 7, 11, 15], 9)
console.log(result)


